doc/user/content/sql/functions/_index.md

```diff
@@ -102,6 +102,10 @@ Operator | Computes
 Operator | Computes
 ---------|---------
 <code>&vert;&vert;</code> | Concatenation
+`~~` | Matches LIKE pattern case sensitively, using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
+`~~*` | Matches LIKE pattern case insensitively (ILIKE), using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
+`!~~` | Does not match LIKE pattern case sensitively, using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
+`!~~*` | Does not match LIKE pattern case insensitively (ILIKE), using [SQL LIKE matching](https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE)
```

Comment:
Are these just synonyms for `LIKE` / `ILIKE`?

Reply:
Yes: https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE